### ISSUE TITLE
Consider adding a UI dashboard

### ISSUE THREAD

**Issue by Razz4780** [metalbear-co/mirrord#2252]
Inspired by [similar solution](https://argo-rollouts.readthedocs.io/en/stable/dashboard/) in Argo.

In OSS it could display active agents, maybe available mirrord targets as well (with some copy-target-name-to-clipboard functionality)

In mirrord for Teams it could be used for session/policy management

**Comment by aviramha** [comment]
I don't think we can have it in the OSS - there's no way to really understand the state when using the OSS (what agent works on what target etc)
I do think it'd be nice when you start a local mirrord session in the OSS you'd have a URL to trace and debug the session - open sockets, fds, logs even maybe then we can add also "chaos" API that allows you to slow down connections, close 'em and stuff like that :)
Another example would be to maybe show the HTTP requests received and the responses, then allow replay
